README.md

```diff
@@ -1,7 +1,7 @@
 <h1 align="center">eslint-plugin-erasable-syntax-only</h1>
 
 <p align="center">
-	ESLint plugin to granularly enforce TypeScript's `erasableSyntaxOnly` flag.
+	ESLint plugin to granularly enforce TypeScript's <a href="https://devblogs.microsoft.com/typescript/announcing-typescript-5-8-rc/#the---erasablesyntaxonly-option"><code>erasableSyntaxOnly</code></a> flag.
```

Comment:
This link was brought in by reverting the file to what's on `main` before re-running `npx create-typescript-app`.